### PR TITLE
Milestone #5 Complete:Integration

### PR DESCRIPTION
**Pull request by Elbie-em**
This branch consists of all of the necessary requirements specified [here](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896095/).

 1. App component 
      * Converted App functional component to class component
      * Initialized state in the constructor
      * Implemented render function that should display Display and ButtonPanel.
      * Implement the handleClick function which takes buttonName as an argument and changes the state based on the 
         result of the calculate function.
      * Passed the total or next value to the Display component to display the current result.
      * Passed handleClick as a click handler property to ButtonPanel
2. ButtonPanel
      * Implemented the handleClick function which takes buttonName as an argument and returns clickHandler from 
         props.
      * Passed the clickHandler property to each button in the panel.
3. Button
      * Implemented the handleClick function which takes buttonName as an argument and returns clickHandler from 
         props.
      * Used the onClick event to attach the handleClick event to the button.
4. Deployed updated complete version to Heroku

The complete version of the application can be accessed on [mminc-calc.herokuapp.com](https://mminc-calc.herokuapp.com/)

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -5,11 +5,31 @@ import ButtonPanel from './ButtonPanel';
 // eslint-disable-next-line no-unused-vars
```
Comment (Forison) [inline src/components/App.js]: - do not modify eslint, kindly do the same in other areas where you disabled the eslint

**Review by Forison — CHANGES_REQUESTED** [review]
## Status:: Change required✖️
# 🙋‍♂️ @Elbie-em 
#### You have done a good job on this one so far, Everything is on point, but for the linter configs, I would have approved the project. I like the UI and the way you handle `%`.🙇‍♂️
* [x] Linters config: Do not make any changes in config files - they represent style guidelines that you share with your team - which is a group of all Microverse students. ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/linter_config.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/54883773/))
- Kindly check my comment below👇
* [x] Kindly address the error logged in the console.
ddd

### That is it, kindly make the changes and submit a re-review💪.
#### Happy coding👨‍💻
Addo Forison

File: src/components/App.js (hunk 1 above)
Comment (Elbie-em) [reply src/components/App.js]: Done

**Review by jofer86 — APPROVED** [review]
## `const PROJECT_STATUS = "APPROVED"` 🚀 🚀 🚀 
## :wave: :wave: Hello! I'm Jorge, and  I will be conducting this code review. Let me get a closer look 🔎 🔍 🔎 🔍 🕵🏼 
- It seems that everything was taken care off, good job!!. Let's move on forward!
--------------------------------------------------------------------
**Rock on**  🤘🏼 
**_Jorge Fernando Rincón Figueroa._** 
Technical Support Engineer.

**Pull request merged**